template/node/Dockerfile

```diff
@@ -33,7 +33,10 @@ COPY function/*.json ./
 RUN npm i --production || :
 
 # Copy in additional function files and folders
-COPY --chown=app:app function/ .
+COPY function .
```

Comment:
I am not convinced that this is the same thing.

@LucasRoesler since you put this on a merge list for me, please could you test/verify the change?